packages/loader/container-loader/src/container.ts

```diff
@@ -1223,6 +1223,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return {
             sequenceNumber: attributes.sequenceNumber,
             version: versionId,
+            quorumSize: this.getQuorum().getMembers().size,
```

Comment:
On a second though, I think it's not worth recording it here. As snapshot maybe already 2 days old, so this info does not carry much weight.